requirements.txt

```diff
@@ -11,5 +11,6 @@ resolvelib
 setuptools<72.0.0
 stevedore
 toml
+tomlkit
```

Comment:
Do we need both toml libraries? We don't have to remove the other one in this PR, but maybe a ticket to do that cleanup?